app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -132,6 +134,8 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
     KIWIX_ROOT = checkWritable(KIWIX_ROOT);
 
+    Log.i("kiwixdownloadservice", "Using Kiwix Root: " + KIWIX_ROOT);
```

Comment:
What's not as obvious to me is *why* we're using KIWIX_ROOT here. I'm for logging the value (particularly as we've lots of challenges identifying where the app should store downloads on various devices). Oh, and for the log message I'd recommend we actually say it's KIWIX_ROOT rather than Kiwix Root. This message is for programmers rather than aimed at end users, and if we use the name of the variable correctly, we can then copy-paste the contents of the message from the log and we know which variable (or const :) ) is being referred to and can locate the relevant code quickly.